user-org-actor/src/main/java/org/sunbird/user/dao/IOSDao.java

```diff
@@ -0,0 +1,19 @@
+package org.sunbird.user.dao;
+
+import org.sunbird.response.Response;
+
+import java.util.Map;
+
+public interface IOSDao {
+
+    Response addEntity(Map<String,Object> entity) throws Exception;
+
+    Response readEntity(Map<String,Object> entity, boolean requireLDResponse) throws Exception;
+
+    Response readEntity(Map<String,Object> entity) throws Exception;
```

Comment:
This method is not required .
* in first method if boolean is false then impl is same as second one , if boolean is true then imple is different , so need to expose one more method.

Reply:
done